network/src/commonMain/kotlin/com/wire/kalium/network/api/model/AccessTokenDTO.kt

```diff
@@ -0,0 +1,25 @@
+package com.wire.kalium.network.api.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+
+@Serializable
+data class AccessTokenDTO (
+    @SerialName("access_token")
+    val value: String,
+    @SerialName("expires_in")
+    val expiresIn: Int,
+    @SerialName("token_type")
+    val tokenType: AccessTokenTypeDTO
```

Comment:
Does the `Type` really matters to us?
Can't it just be a `String` and we try to use it like that?

Reply:
No, not really needed, I will change it to a string for simplicity